src/app/examples/badge-example/badge-example.component.tns.html

```diff
@@ -0,0 +1,15 @@
+<StackLayout>
+  <GridLayout rows="auto" columns="auto, auto">
+    <Label class="h1" text="Badge" col="0" row="0" colspan="2"></Label>
+    <FlexBoxLayout col="1" row="0" flexDirection="row" alignItems="center" justifyContent="flex-end" marginRight="-10">
+      <kirby-badge colortype="danger" text="2"></kirby-badge>
+    </FlexBoxLayout>
+  </GridLayout>
+
+  <GridLayout rows="auto" columns="auto, auto">
+    <Label class="h1" text="Other Badge" col="0" row="0" colspan="2"></Label>
+    <FlexBoxLayout col="1" row="0" flexDirection="row" alignItems="center" justifyContent="flex-end" marginRight="-10">
+      <kirby-badge colortype="primary" text="Text ex."></kirby-badge>
+    </FlexBoxLayout>
+  </GridLayout>
+</StackLayout> 
```

Comment:
Consider adding example with icon as well:
```suggestion
<GridLayout rows="auto" columns="auto, auto">
    <Label class="h1" text="Other Badge" col="0" row="0" colspan="2"></Label>
    <FlexBoxLayout col="1" row="0" flexDirection="row" alignItems="center" justifyContent="flex-end" marginRight="-10">
      <kirby-badge colortype="primary">
        <kirby-icon name="close></kirby-icon>
      </kirby-badge>
    </FlexBoxLayout>
  </GridLayout>
</StackLayout> 
```